src/content/advanced/iam-roles-for-service-accounts/index.md

```diff
@@ -39,23 +39,23 @@ Please first ensure having the following permissions added on the [`GiantSwarmAW
       ],
       "Resource": "*"
   },
-...
   {
       "Effect": "Allow",
       "Action": [
           "s3:PutObjectAcl"
       ],
       "Resource": "arn:aws:s3:::*-g8s-*"
   },
- ...
-```
-
-With AWS release v18.0.0 we additionally need permissions for managing Cloudfront (**only non-China regions**). By upgrading to this AWS release the S3 bucket which contains the OIDC configuration and public keys will be protected and direct traffic will be blocked. We only allow traffic through Cloudfront.
```

Comment:
why did we remove this part?

Reply:
I think its confusing to have two versions, now it would be 3 versions of the role...

As we just add more things to the role customers should just make sure all of them are in place, or you think its better to keep sepparate?

Reply:
Makes sense👍